src/service/resources-impl.js

```diff
@@ -169,6 +173,7 @@ export class Resources {
     });
     this.viewport_.onScroll(() => {
       this.lastScrollTime_ = timer.now();
+      this.framerate_.collect();
```

Comment:
The operation here is extremely cheap. Doesn't do anything except once per frame.